Sources/TSCBasic/Path.swift

```diff
@@ -680,11 +679,13 @@ private struct UNIXPath: Path {
 
         // If the result is empty, return `.`, otherwise we return it as a string.
         self.init(string: result.isEmpty ? "." : result)
```

Comment:
Removing the use of `PathCanonicalizeW` is fine, but this no longer canonicalizes all paths.  Please handle UNC paths and drive roots.